File: AnimalShelterAPI/AnimalShelterAPI/Models/Animal.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace AnimalShelterAPI.Models
{
    public class Animal
    {
        public int ID { get; set; }
        public DateTime AdmissionDate { get; set; }
        public DateTime? MicrochipIntegrationDate { get; set; }
        public DateTime? VaccinationDate { get; set; }
        public string AdmissionCity { get; set; }
        public string AdmissionRegion { get; set; }
        public AnimalType AnimalType { get; set; }
        public Gender Gender { get; set; } 
        public Fur Fur { get; set; }
        public string SpecialTags { get; set; }
        public string HealthCondition { get; set; }
        public string AdmissionOrganisationContacts { get; set; }
        public Status Status { get; set; }
        public DateTime? StatusDate { get; set; }
    }
}

File: WebApp/src/app/models/animal.ts
import {Fur} from '@app/models/fur';

export class Animal {
  id: number;
  specialID: string;
  admissionDate: Date;
  microchipIntegrationDate: Date;
  vaccinationDate: Date;
  admissionCity: string;
  admissionRegion: string;
  animalType: number;
  birthday: Date;
  months: number;
  fur: Fur;
  specialTags: string;
  healthCondition: string;
  admissionOrganisationContacts: string;
}
